Sources/PackageFingerprint/FilePackageFingerprintStorage.swift

```diff
@@ -179,12 +229,28 @@ private enum StorageModel {
     }
 }
 
-extension PackageIdentity {
-    var fingerprintFilename: String {
+protocol FingerprintReference {
+    var fingerprintsFilename: String { get }
+}
+
+extension PackageIdentity: FingerprintReference {
+    var fingerprintsFilename: String {
         "\(self.description).json"
     }
 }
 
+extension PackageReference: FingerprintReference {
+    var fingerprintsFilename: String {
+        guard case .remoteSourceControl(let sourceControlURL) = self.kind else {
+            fatalError("Package kind [\(self.kind)] does not support fingerprints")
```

Comment:
this will crash the program, if called incorrectly. would it be safe to make it into a throwing function?